src/containerapp/azext_containerapp/_params.py

```diff
@@ -123,6 +123,7 @@ def load_arguments(self, _):
         c.argument('workload_profile_name', options_list=['--workload-profile-name', '-w'], help="Name of the workload profile to run the app on.")
         c.argument('secret_volume_mount', help="Path to mount all secrets e.g. mnt/secrets")
         c.argument('termination_grace_period', type=int, options_list=['--termination-grace-period', '--tgp'], help="Duration in seconds a replica is given to gracefully shut down before it is forcefully terminated. (Default: 30)")
+        c.argument('source', help="Local directory path containing the application source and Dockerfile for building the container image. Preview: If no Dockerfile is present, a container image is generated using buildpacks. If Docker is not running or buildpacks cannot be used, Oryx will be used to generate the image. See the supported Oryx runtimes here: https://github.com/microsoft/Oryx/blob/main/doc/supportedRuntimeVersions.md.")
```

Comment:
add `is_preview=True` to mark the parameter as Preview first

Reply:
Since the `create` and `update` commands have already GA'd, confirmed with PM that this should not be in preview.

Reply:
Apologies, I misunderstood. Added `is_preview=True` for `--source` and `--repo`.

Reply:
Add `is_preview=True` for --repo and --source in command `az containerapp create/update`, it is expected because they are new preview argument for command `az containerapp create/update`， nothing to do with `az containerapp up`

Reply:
Thanks for clarifying @Greedygre!